scss/forms/_select.scss

```diff
@@ -20,7 +20,8 @@ $select-radius: $global-radius !default;
 
 @mixin form-select {
   $height: ($input-font-size + ($form-spacing * 1.5) - rem-calc(1));
-
+  
+  box-sizing: border-box;
```

Comment:
This is a reset property for specifically the `<select>` tag. I think it would be more appropriate in Global properties: [scss/_global.scss:169](https://github.com/ncoden/foundation-sites/blob/5573453df250bdf0ce1dec239675cc0375e3e152/scss/_global.scss#L169)

```scss
// Force box-sizing for Chrome
// https://bugs.chromium.org/p/chromium/issues/detail?id=589475
box-sizing: border-box;
```